packages/terra-clinical-header/src/Header.jsx

```diff
@@ -16,46 +16,61 @@ const propTypes = {
   children: PropTypes.element,
 
   /**
-   * Content to be displayed at the start of the header, placed before the title
+   * Content to be displayed at the end of the header
+   * The element passed as endContent will be wrapped in a div with flex attributes.
    */
-  startContent: PropTypes.element,
+  endContent: PropTypes.element,
+
+  /*
+  * IDs should be used with ‘aria-labelledby’ to associate headings with the corresponding page region.
+  */
+  id: PropTypes.string,
 
   /**
-   * Text to be displayed as the title in the header bar
+   * A Boolean indicating if element is a subheader.
    */
-  title: PropTypes.string,
+  isSubheader: PropTypes.bool,
 
   /**
-   * Content to be displayed at the end of the header
-   * The element passed as endContent will be wrapped in a div with flex attributes.
+   * Sets the heading level. One of `1`, `2`, `3`, `4`, `5`, `6`. This helps screen readers to announce appropriate heading levels.
+   * Changing 'level' will not visually change the style of the content.
    */
-  endContent: PropTypes.element,
+  level: PropTypes.oneOf([1, 2, 3, 4, 5, 6]),
```

Comment:
i am not sure, does this values more sense for consumers. if we mention it as strings ? "One", "two" etc

cc. @neilpfeiffer

Reply:
I think using number is ok because they really are `h1`, `h2`, etc. If we used strings we would have to map them back to a number to be able to do this:

> const HeaderElement = `h${level}`;